include/mbedtls/platform.h

```diff
@@ -196,11 +200,12 @@ int mbedtls_platform_win32_snprintf( char *s, size_t n, const char *fmt, ... );
 extern int (*mbedtls_snprintf)( char * s, size_t n, const char * format, ... );
 
 /**
- * \brief   Set your own snprintf function pointer
+ * \brief   This function allows configuring a custom \c snprintf function 
+ *           pointer.
```

Comment:
Style: Superfluous space. Feel free to amend the previous commit if you like.

Reply:
fixed